### PR TITLE
Update getting started guides for Android

### PR DESCRIPTION
**Pull request by OlegRyz**
- Remove a paragraph with THEOplayer 4.12.x differences
- Convert examples to Kotlin
- Update screenshots using Kotlin version

### CODE REVIEW

File: theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx [hunk 1]
```diff
@@ -0,0 +1,476 @@
...
+```xml
+<uses-permission android:name="android.permission.INTERNET" />
+```
+
+More information regarding the license can be found [here](../../../how-to-guides/12-license/00-introduction.md).
+
+## Add THEOplayer view to your application
+
+THEOplayer can be added to an application either through a layout (i.e. XML) or programmatically through the constructor API. This guide uses the layout approach. Refer to [Using the constructor API](https://github.com/THEOplayer/theoplayer-sdk-android#b-using-the-constructor-api)
+if you want to use the latter approach. This approach also allows you to configure the license string through a programmatic configuration instead of the `AndroidManifest.xml`.
```
Comment (MattiasBuelens) [inline theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx:99]: We should move the "Using the constructor API" section into this article too.

File: theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx [hunk 2]
```diff
@@ -0,0 +1,476 @@
...
+</com.theoplayer.android.api.THEOplayerView>
+```
+
+![](../../../assets/img/android-getting-started-14_2.png)
+
+## Use THEOplayer API in your application
+
+Now in the `MainActivity` we can retrieve the `THEOplayerView` by the `id` we set earlier using the `findViewById` method.
+
+#### 1. Get the THEOplayer instance
```
Comment (MattiasBuelens) [inline theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx:124]: These headings are nested one level too deep.
```suggestion
### 1. Get the THEOplayer instance
```
Same for the other numbered steps.

File: theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx [hunk 3]
```diff
@@ -0,0 +1,476 @@
...
+            txtPlayStatus.text = "Playing"
+        }
+
+        theoPlayerView.player.addEventListener(PlayerEventTypes.PAUSE) {
+            txtPlayStatus.text = "Paused"
+        }
+
+        theoPlayerView.player.addEventListener(PlayerEventTypes.TIMEUPDATE) { timeUpdateEvent ->
+        txtTimeUpdate.text = timeUpdateEvent.currentTime.toString()
+        }
```
Comment (MattiasBuelens) [inline theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx:292]: Fix indentation.
```suggestion
        theoPlayerView.player.addEventListener(PlayerEventTypes.TIMEUPDATE) { timeUpdateEvent ->
            txtTimeUpdate.text = timeUpdateEvent.currentTime.toString()
        }
```

File: theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx [hunk 4]
```diff
@@ -0,0 +1,476 @@
...
+This guide focuses on creating an Android application from scratch and integrating the THEOplayer Android SDK.
+
+- You have learned how to interact with the player via the available APIs in Java,
+- you have registered player events,
+- and you have learned how to enable automatic orientation detection.
+
+You can find various samples for different use cases on [GitHub](https://github.com/THEOplayer/samples-android-sdk/tree/master).
+
+You may also find the readme at [https://github.com/THEOplayer/theoplayer-sdk-android](https://github.com/THEOplayer/theoplayer-sdk-android) useful.
+
```
Comment (MattiasBuelens) [inline theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx:473]: I was hoping we could move all of the useful information from that readme into our documentation website, so we would no longer need to link to it.

**Review by MattiasBuelens — COMMENTED** [review]
Looking pretty good so far! 👍

File: theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx (hunk 1 above)
Comment (OlegRyz) [reply theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx]: Done

File: theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx (hunk 2 above)
Comment (OlegRyz) [reply theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx]: Done

File: theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx (hunk 3 above)
Comment (OlegRyz) [reply theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx]: Done

File: theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx (hunk 4 above)
Comment (OlegRyz) [reply theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx]: I checked that readme and I think the all useful info from there is already here. So, indeed, we don't need to include this link.

File: theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx [hunk 5]
```diff
@@ -0,0 +1,508 @@
...
+![](../../../../theoplayer/assets/img/android-getting-started-14_2.png)
+
+
+### Adding THEOplayerView programmatically (Constructor API)
+
+In the Activity/Fragment, you can use the [constructor API](https://docs.theoplayer.com/api-reference/android/index.html?com/theoplayer/android/api/THEOplayerView.html) to create a THEOplayer instance
+by passing along a context and a [`THEOplayerConfig`](https://docs.theoplayer.com/api-reference/android/index.html?com/theoplayer/android/api/THEOplayerConfig.html).
+
+In this `THEOplayerConfig`, you must configure your license string, as demonstrated in the snippet below.
```
Comment (hovig-theo) [inline theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx:128]: "must" is wrong in this context.
Users have the option to either set the license in the AndroidManifest or through the THEOplayerConfig API. Even when using the constructor, they could set the license in the AndroidManifest.

File: theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx [hunk 6]
```diff
@@ -0,0 +1,508 @@
...
+            }
+
+            // ...
+        }
+        ```
+        </TabItem>
+        <!-- prettier-ignore-end -->
+    </Tabs>
+
+If you would like to customize the Fullscreen Activity, you can take a look at our sample [here](https://github.com/THEOplayer/samples-android-sdk/tree/master/full-screen-handling).
```
Comment (hovig-theo) [inline theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx:474]: This is part of the fullscreen section above.

File: theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx [hunk 7]
```diff
@@ -0,0 +1,508 @@
...
+
+### Rotated fullscreen
+
+![](../../../assets/img/android-getting-started-screenshot-4.png)
+
+## Summary
+
+This guide focuses on creating an Android application from scratch and integrating the THEOplayer Android SDK.
+
+- You have learned how to interact with the player via the available APIs in Java,
```
Comment (hovig-theo) [inline theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx:500]: Nitpcik: Kotlin too

File: theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx [hunk 8]
```diff
@@ -0,0 +1,508 @@
...
+- you have registered player events,
+- and you have learned how to enable automatic orientation detection.
+
+You can find various samples for different use cases on [GitHub](https://github.com/THEOplayer/samples-android-sdk/tree/master).
+
+You can find interesting features of THEOplayer on our [demo](https://www.theoplayer.com/theoplayer-demo-overview) page.
```
Comment (hovig-theo) [inline theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx:506]: This demo page is relevant for the Web SDK only

File: theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx (hunk 6 above)
Comment (OlegRyz) [reply theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx]: Moved

File: theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx (hunk 7 above)
Comment (OlegRyz) [reply theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx]: yup, thanks!

File: theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx (hunk 8 above)
Comment (OlegRyz) [reply theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx]: I removed it

File: theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx [hunk 9]
```diff
@@ -125,7 +125,7 @@ At least the `id`, `width` and `height` attributes need to be specified on the n
 In the Activity/Fragment, you can use the [constructor API](https://docs.theoplayer.com/api-reference/android/index.html?com/theoplayer/android/api/THEOplayerView.html) to create a THEOplayer instance
 by passing along a context and a [`THEOplayerConfig`](https://docs.theoplayer.com/api-reference/android/index.html?com/theoplayer/android/api/THEOplayerConfig.html).
 
-In this `THEOplayerConfig`, you must configure your license string, as demonstrated in the snippet below.
+In this `THEOplayerConfig`, you could configure your license string, as demonstrated in the snippet below.
```
Comment (MattiasBuelens) [inline theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx:128]:
```suggestion
In this `THEOplayerConfig`, you can configure your license string, as demonstrated in the snippet below.
```

File: theoplayer_versioned_docs/version-v6/getting-started/01-sdks/02-android/00-getting-started.mdx [hunk 10]
```diff
@@ -125,7 +125,7 @@ At least the `id`, `width` and `height` attributes need to be specified on the n
 In the Activity/Fragment, you can use the [constructor API](https://docs.theoplayer.com/api-reference/android/index.html?com/theoplayer/android/api/THEOplayerView.html) to create a THEOplayer instance
 by passing along a context and a [`THEOplayerConfig`](https://docs.theoplayer.com/api-reference/android/index.html?com/theoplayer/android/api/THEOplayerConfig.html).
 
-In this `THEOplayerConfig`, you must configure your license string, as demonstrated in the snippet below.
+In this `THEOplayerConfig`, you could configure your license string, as demonstrated in the snippet below.
```
Comment (MattiasBuelens) [inline theoplayer_versioned_docs/version-v6/getting-started/01-sdks/02-android/00-getting-started.mdx:128]:
```suggestion
In this `THEOplayerConfig`, you can configure your license string, as demonstrated in the snippet below.
```

File: theoplayer_versioned_docs/version-v7/getting-started/01-sdks/02-android/00-getting-started.mdx [hunk 11]
```diff
@@ -125,7 +125,7 @@ At least the `id`, `width` and `height` attributes need to be specified on the n
 In the Activity/Fragment, you can use the [constructor API](https://docs.theoplayer.com/api-reference/android/index.html?com/theoplayer/android/api/THEOplayerView.html) to create a THEOplayer instance
 by passing along a context and a [`THEOplayerConfig`](https://docs.theoplayer.com/api-reference/android/index.html?com/theoplayer/android/api/THEOplayerConfig.html).
 
-In this `THEOplayerConfig`, you must configure your license string, as demonstrated in the snippet below.
+In this `THEOplayerConfig`, you could configure your license string, as demonstrated in the snippet below.
```
Comment (MattiasBuelens) [inline theoplayer_versioned_docs/version-v7/getting-started/01-sdks/02-android/00-getting-started.mdx:128]:
```suggestion
In this `THEOplayerConfig`, you can configure your license string, as demonstrated in the snippet below.
```

**Review by hovig-theo — APPROVED** [review]

**Review by MattiasBuelens — APPROVED** [review]

**Pull request merged**